src/Parser/Hunter/Marksmanship/Modules/Talents/PatientSniper/PatientSniperDetails.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Tab from 'Main/Tab';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber, formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+
+import PatientSniperBreakdown from "./PatientSniperBreakdown";
+import PatientSniperTracker from "./PatientSniperTracker";
+
+const VULNERABLE_BONUS = 0.3;
+const UNERRING_ARROWS_BONUS_PER_RANK = 0.03;
+
+class PatientSniperDetails extends Module {
+  static dependencies = {
+    patientSniperTracker: PatientSniperTracker,
+    combatants: Combatants,
+    abilityTracker: AbilityTracker,
+  };
+
+  hasPiercingShot = false;
+  vulnerableModifier = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.PATIENT_SNIPER_TALENT.id);
+    this.hasPiercingShot = !!this.combatants.selected.hasTalent(SPELLS.PIERCING_SHOT_TALENT.id);
+    const unerringArrowsRank = this.combatants.selected.traitsBySpellId[SPELLS.UNERRING_ARROWS_TRAIT.id];
+    this.vulnerableModifier = VULNERABLE_BONUS + unerringArrowsRank * UNERRING_ARROWS_BONUS_PER_RANK;
+  }
+
+  get bonusAimedDamage() {
+    return this.patientSniperTracker.patientSniper[SPELLS.AIMED_SHOT.id].bonusDmg;
+  }
+  get bonusPiercingDamage() {
+    return this.patientSniperTracker.patientSniper[SPELLS.PIERCING_SHOT_TALENT.id].bonusDmg;
+  }
+  get bonusDamage() {
+    return this.bonusAimedDamage + this.bonusPiercingDamage;
+  }
+
+  suggestions(when) {
```

Comment:
This should be updated to be 0.09, 0.08, 0.07 for minor, average and major as I mentioned, looking at some top logs they seem hover between 9 and 10% increased damage, and the moment it starts dropping lower it indicates a wrong rotation - as PS damage should be a damage increase from purely playing rotationally correctly (apply vulnerable --> generate focus --> aimed shotx2 late in the vuln window) 

These values are 'only' accurate on more ST focused encounters, but the moment we enter AoE , it becomes way less important to care about Patient Sniper, which then leads to automatically utilising it less - however I still think the recommended values should be for ST encounters, possibly include the fact that it may look bad on AoE, it could be because Patient Sniper isn't as important there.

Reply:
Ok I'll update the thresholds when I can :-)